### DISCUSSION TITLE
Quoted reclouts - engagement activity mechanic

### DISCUSSION THREAD

**Discussion by 0xNordian** [deso-protocol/dips#73] [Medium CIPs]
cip: TBA
title: Quoted reclouts - engagement activity mechanic
author: Nordian ( @N0rdian)
discussions-url: N/A
created: 2021-07-15
-------------------------

### One-Line Description
Reclouts engagement activity tally (_likes, reclout, diamonds_) should be assigned to the original content creator when a post gets shared as a quoted reclout.

-------------------------
### Explanation and Motivation
Currently, when a post gets reclouted with a quote, all the resulting engagement activity will go to the user that has reclouted with a quote and not to the original post creator. IMHO, when you reclout a piece of content, you are recognising the value of that piece of content is creating and want to share it for:

1. Share the value with the community.
2. Recognise the value created by a content creator and support them by spreading the word.

So I would like to propose that all quoted reclouts engagement activity tally should be assigned to the original content creator.

This change could incentivise the creation of great content, especially from 'smaller' creators. It also avoids concentrating all the engagement on the 'biggest' creators, which in the long run will collect the majority of the activity engagement of a reclout if we continue in the current model.

If a user wants to create value on top of the original post and obtain the result of the activity engagement, they should do it through a reply to the original post IMHO.

-------------------------
### Specification
I cannot share technical specifications as I don't have a technical background.

-------------------------
### Backwards Compatibility
I don't think this proposed change will have 'backwards compatibility' issues.

-------------------------
### Test Cases
I cannot share technical specifications as I don't have a technical background.

-------------------------
### Security Considerations
I don't think there are any major security issues to be considered in this proposal.

-------------------------
Thanks for taking the time to consider this proposal.

**Comment by tijno** [comment]
I think this would be fairly easy to implement here:

https://github.com/bitclout/frontend/blob/main/src/app/feed/feed-post/feed-post.component.ts#L31

I agree that adding considerable value to a post could be done by replying, and then reclouting your own reply. Rather then quote reclouting the original.

The only uncertainty i have is - what to do with replies?

Currently replies, like likes and diamonds, go to the post & poster quoting the original.

What should happen to replies? Should they remain on the post quoting the other? Or should they also show in the original comment feed?

Also - what about quote reclout of a quote  reclout?  Currently a quote of a quote will show upto 2 quoted posts.

**Reply by 0xNordian** [reply]
Thanks for the feedback.

> Currently replies, like likes and diamonds, go to the post & poster quoting the original.

I'm not sure if I understand this one. 
My understanding is that the engagement activity of the replies of a quoted reclout goes to the person that has done the reply and not for the person that has done the quoted reclout, which as I mentioned before I think is the proper way to collect engagement activity by creating value in a post. Apologies if I'm not understanding you correctly.

Regarding the number of replies of a quoted reclout, that is a very interesting point and it has many considerations. I will take some time to reply to this.

> What should happen to replies? Should they remain on the post quoting the other? Or should they also show in the original comment feed?

I'm inclined to say _'they remain on the post quoting the other'_ but I will come back to this later after thinking more about your feedback and see the feedback of other users.

Thanks as always for your insights @tijno.

**Reply by tijno** [reply]
> I'm not sure if I understand this one.

It was related to the point about - what happens to the replies. Which you suggested "they remain on the post quoting the other" which i agree with.

**Comment by ShhubhamA** [comment]
I think we may have a solution if there is a small diamond and like button for the original post too in the quoted posts!!

I think curation is cool and i think a user could add value to the content by quoting him and it should be upto the one who diamonds to decide what he likes .
Thanks
